src/org/mozilla/javascript/NativeStringIterator.java

```diff
@@ -7,7 +7,7 @@
 package org.mozilla.javascript;
 
 public final class NativeStringIterator extends ES6Iterator {
-    private static final long serialVersionUID = 1L;
+    private static final long serialVersionUID = 5890115206182483238L;
```

Comment:
Technically this breaks backward compatibility. Rhino is used in a lot of places and we never know. Can you take this one out?

Reply:
Yes, sir